src/components/Applications/ApplicationList.vue

```diff
@@ -0,0 +1,61 @@
+<template>
+  <q-card>
+    <ApplicationItem
+      v-for="a in pendingApplications"
+      :key="a.id"
+      :application="a"
+      @accept="$emit('forwardAccept', arguments[0])"
```

Comment:
Usually we keep the same event name when forwarding events, so `$emit('accept', arguments[0])` would be better.